client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/GraphQlClientBuilder.java

```diff
@@ -1,6 +1,7 @@
 package io.smallrye.graphql.client.typesafe.api;
 
 import java.net.URI;
+import java.net.URL;
```

Comment:
I don't see where URL is used in this class.  Maybe this was accidentally added?  Or was there an intended `endpoint(URL url)` method?

Reply:
JAX-RS uses URI; URL was a mistake. Thanks!